packages/wallets/walletconnect/src/adapter.ts

```diff
@@ -33,9 +33,30 @@ export enum WalletConnectRPCMethod {
     signMessage = 'solana_signMessage',
 }
 
+const getWalletConnectParams = (chainId: WalletConnectChainID, pairingTopic?: string): EngineTypes.ConnectParams => ({
+    requiredNamespaces: {
+        solana: {
+            chains: [chainId],
+            methods: [WalletConnectRPCMethod.signTransaction, WalletConnectRPCMethod.signMessage],
+            events: [],
+        },
+    },
+    pairingTopic,
+});
+
+const getChainId = (network: WalletAdapterNetwork): WalletConnectChainID => {
+    switch (network) {
+        case WalletAdapterNetwork.Mainnet:
+            return WalletConnectChainID.Mainnet;
+        case WalletAdapterNetwork.Devnet:
+        default:
+            return WalletConnectChainID.Devnet;
```

Comment:
In general the adapters default to mainnet. But I'm wondering if we want to silently just accept testnet or something if it will actually not work. Seems better to fail fast?